core/templates/components/ck-editor-helpers/ck-editor-copy-content-service.ts

```diff
@@ -17,11 +17,11 @@
  *               ck editor
  */
 
-import { downgradeInjectable } from '@angular/upgrade/static';
 import { Injectable, EventEmitter } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
 
-import { HtmlEscaperService } from 'services/html-escaper.service';
 import { Subscription } from 'rxjs';
+import { HtmlEscaperService } from 'services/html-escaper.service';
```

Comment:
Why is this together with `Subscription`?

Reply:
@vojtechjelinek All of these are grouped in this order ["builtin", "external", "parent", "sibling", "index"]. You can see the configuration here (https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/order.md)

Reply:
Sure but then it seems that `rxjs` should belong to builtin or external? An stuff from `services` should belong to internal.